float64 X
float64 Y
float64 phi
---
bool reached_position